src/components/scene/canvas.js

```diff
@@ -29,6 +29,8 @@ module.exports.Component = register('canvas', {
       canvas.style.height = data.height + '%';
       canvas.style.width = data.width + '%';
       scene.appendChild(canvas);
+    } else {
+      canvas.dataset.aframeProvided = true;
```

Comment:
Ah, I think the confusion is we're thinking "provided by A-Frame", and you're thinking "provided by user"